Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/TaskRecyclerViewAdapter.java

```diff
@@ -0,0 +1,124 @@
+package pg.autyzm.friendly_plans;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import database.entities.TaskTemplate;
+
+
+/**
+ * Created by Mateusz on 2017-03-08.
+ */
+
+public class TaskRecyclerViewAdapter extends RecyclerView.Adapter<TaskRecyclerViewAdapter.TaskListViewHolder> {
+
+
+    private List<TaskTemplate> taskItemList;
+    private TaskItemClickListener taskItemClickListener;
+    private int ICON_PLACEHOLDER_ID = R.drawable.ic_placeholder;
+
+    @Override
+    public TaskListViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_task, parent, false);
+        return new TaskListViewHolder(view, taskItemClickListener);
+    }
+
+    @Override
+    public void onBindViewHolder(TaskListViewHolder holder, int position) {
+
+        if (taskItemList != null || taskItemList.size() != 0) {
+
+            TaskTemplate taskItem = taskItemList.get(position);
+            holder.taskName.setText(taskItem.getName());
+            //condition only for mocking purposes -> needs to be changed later
```

Comment:
I think we should keep checking is picture/sound/... set because some tasks won't have them. I have some idea - maybe to make this more readable we can extract setting picture/sound/... to helper methods?